### PR TITLE
ci: Update build timeouts

### PR DESCRIPTION
**Pull request by jkleinsc**
#### Description of Change
This PR updates the native_mksnapshot build to have a longer timeout for arm builds.
Also, this PR reverts #16479 so that arm releases will be built.

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: no-notes

### CODE REVIEW

File: script/ci-release-build.js [hunk 1]
```diff
@@ -136,11 +134,9 @@ function buildCircleCI (targetBranch, options) {
...
 
 async function buildVSTS (targetBranch, options) {
-  // FIXME(codebytere): re-enable
-  // if (options.armTest) {
-  //   assert(vstsArmJobs.includes(options.job), `Unknown VSTS CI arm test job name: ${options.job}. Valid values are: ${vstsArmJobs}.`)
-  // }
-  if (options.job) {
+  if (options.armTest) {
+    assert(vstsArmJobs.includes(options.job), `Unknown VSTS CI arm test job name: ${options.job}. Valid values are: ${vstsArmJobs}.`)
+  } else if (options.job) {
```
Comment (MarshallOfSound) [inline script/ci-release-build.js]: This is a slight change in behavior from the commented out code

Comment (jkleinsc) [reply script/ci-release-build.js]: Huh?  I just reverted #16479

Comment (jkleinsc) [reply script/ci-release-build.js]: options.armTest and options.job are mutually exclusive

**Comment by jkleinsc** [comment]
Merging as pending CI isn't related to circleCI change

**Pull request merged**